ui/eureka/eureka.js

```diff
@@ -2139,6 +2620,40 @@ class EurekaTracker {
     }
   }
 
+  OnCE(e) {
+    // Upon entering Eureka we usually receive the CE info before
+    // this.zoneInfo is loaded, so lets store the events until we're
+    // able to process them.
+
+    if (!this.zoneInfo) {
+      this.CEQueue.push(e);
+      return;
+    }
+
+    const handleEvent = (handler) => {
+      for (const key of this.nmKeys) {
+        const nm = this.nms[key];
+        if (e.detail.data.ceKey == nm.ceKey)
```

Comment:
did the `===` get reverted? :eyes:

Reply:
This handleEvent function definition should also go to a higher scope if possible, it doesn't make much sense to create a new function object every CE event.
Perhaps this should be a class method?